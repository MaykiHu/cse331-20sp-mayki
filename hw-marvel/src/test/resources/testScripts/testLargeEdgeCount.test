#Tests for the edgeCount of a large dataset, from marvel.tsv
#marvel.tsv should be a valid file under the marvel data folder

LoadGraph largeHeroes marvel.tsv
GetEdgeCount largeHeroes