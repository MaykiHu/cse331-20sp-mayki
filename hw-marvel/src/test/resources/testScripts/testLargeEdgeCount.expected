#Tests for the edgeCount of a large dataset, from marvel.tsv
#marvel.tsv should be a valid file under the marvel data folder

loaded graph largeHeroes
Number of edges in largeHeroes is: 1139314